tests/test_2023_04_25.py

```diff
@@ -0,0 +1,145 @@
+    NodeOperatorRewardAddressSetItem,
+)
+from utils.test.event_validators.easy_track import (
+    validate_motions_count_limit_changed_event
+)
+
+def test_vote(
+    helpers,
+    accounts,
+    vote_id_from_env,
+    bypass_events_decoding,
+    ldo_holder,
+    dao_voting,
+    node_operators_registry,
+    easy_track
+):
+
+    CertusOne_Jumpcrypto_id = 1
+    CertusOne_Jumpcrypto_name_before = "Certus One"
+    CertusOne_Jumpcrypto_name_after = "jumpcrypto"
+
+    ConsenSysCodefi_Consensys_id = 21
+    ConsenSysCodefi_Consensys_name_before = "ConsenSys Codefi"
+    ConsenSysCodefi_Consensys_name_after = "Consensys"
+
+    SkillZ_Kiln_id = 8
+    SkillZ_Kiln_name_before = "SkillZ"
+    SkillZ_Kiln_name_after = "Kiln"
+    SkillZ_Kiln_address_before = "0xe080E860741b7f9e8369b61645E68AD197B1e74C"
+    SkillZ_Kiln_address_after = "0xD6B7d52E15678B9195F12F3a6D6cb79dcDcCb690"
+
+    motionsCountLimit_before = 12
+    motionsCountLimit_after = 20
+
+    # get NO's data before
+    CertusOne_Jumpcrypto_data_before = node_operators_registry.getNodeOperator(CertusOne_Jumpcrypto_id, True)
+    ConsenSysCodefi_Consensys_data_before = node_operators_registry.getNodeOperator(ConsenSysCodefi_Consensys_id, True)
+    SkillZ_Kiln_data_before = node_operators_registry.getNodeOperator(SkillZ_Kiln_id, True)
+
+    # check names before
+    assert CertusOne_Jumpcrypto_data_before[1] == CertusOne_Jumpcrypto_name_before, "Incorrect NO#1 name before"
+    assert ConsenSysCodefi_Consensys_data_before[1] == ConsenSysCodefi_Consensys_name_before, "Incorrect NO#21 name before"
+    assert SkillZ_Kiln_data_before[1] == SkillZ_Kiln_name_before, "Incorrect NO#8 name before"
+
+    # check SkillZ_Kiln reward address before
+    assert SkillZ_Kiln_data_before[2] == SkillZ_Kiln_address_before, "Incorrect NO#8 reward address before"
+
+    # check Easy Track motions count limit before
+    easy_track.motionsCountLimit() == motionsCountLimit_before, "Incorrect motions count limit before"
+
+    ##
+    ## START VOTE
+    ##
+    vote_id = vote_id_from_env or start_vote({"from": ldo_holder}, silent=True)[0]
+
+    tx: TransactionReceipt = helpers.execute_vote(
+        vote_id=vote_id, accounts=accounts, dao_voting=dao_voting, skip_time=3 * 60 * 60 * 24
+    )
+
+    # get NO's data before
+    CertusOne_Jumpcrypto_data_after = node_operators_registry.getNodeOperator(CertusOne_Jumpcrypto_id, True)
+    ConsenSysCodefi_Consensys_data_after = node_operators_registry.getNodeOperator(ConsenSysCodefi_Consensys_id, True)
+    SkillZ_Kiln_data_after = node_operators_registry.getNodeOperator(SkillZ_Kiln_id, True)
+
+    # compare NO#1 (CertusOne -> Jumpcrypto) data before and after
+    assert CertusOne_Jumpcrypto_data_before[0] == CertusOne_Jumpcrypto_data_after[0] # active
+    assert CertusOne_Jumpcrypto_name_after == CertusOne_Jumpcrypto_data_after[1] # name
+    assert CertusOne_Jumpcrypto_data_before[2] == CertusOne_Jumpcrypto_data_after[2] # rewardAddress
```

Comment:
I'd prefer introducing the constants for indexes `0, 1, 2, 3, 4` to make it more reader-friendly
Otherwise, can use a special structures (the hard way, huh)